### PR TITLE
leave summary page created

### PR DESCRIPTION
**Pull request by reactexcel**

### CODE REVIEW

File: src/components/leavesSummary/ViewLeavesSummary.jsx [hunk 1]
```diff
@@ -1,209 +1,162 @@
...
-
-        let show_text = ""
-
-        if( day.day_type == 'HALF_DAY' ){
-          check_class = "b-warn"
-          show_text = day.day_type + ' / ' + "Timings : " + day.in_time +' - ' + day.out_time + ' / Total Time : ' + day.total_time
+  _getPendingPunchingDays(p) {
+    return _.map(p, (day, key) => {
+      let check_class = "b-success"
+      if (day.day_type == 'WORKING_DAY' || day.day_type == 'LEAVE_DAY') {
```
Comment (upexcel) [inline src/components/leavesSummary/ViewLeavesSummary.jsx]: put in config

File: src/containers/admin/leavesSummary.jsx [hunk 2]
```diff
@@ -22,41 +25,28 @@ class LeavesSummary extends React.Component {
...
-    //         let firstUser = props.usersList.users[0]
-    //         let defaultUserId = firstUser.user_Id
-
-    //     }
-    // }
+    //window.scrollTo(0, 0);
+    if (props.logged_user.logged_in == -1) {
+      this.props.router.push('/logout');
+    } else {
+      if (props.logged_user.role == 'Admin' || props.logged_user.role == 'Guest' || props.logged_user.role == 'HR') {
```
Comment (upexcel) [inline src/containers/admin/leavesSummary.jsx]: this should come from config

File: src/components/leavesSummary/ViewLeavesSummary.jsx (hunk 1 above)
Comment (reactexcel) [inline src/components/leavesSummary/ViewLeavesSummary.jsx]: ohk sir

File: src/containers/admin/leavesSummary.jsx (hunk 2 above)
Comment (reactexcel) [inline src/containers/admin/leavesSummary.jsx]: ohk sir

**Pull request merged**